homeassistant/components/sonos/media_player.py

```diff
@@ -431,7 +431,9 @@ def _play_favorite_by_name(self, name: str) -> None:
         fav = [fav for fav in self.speaker.favorites if fav.title == name]
 
         if len(fav) != 1:
-            return
+            raise ServiceValidationError(
+                f'Could not find a Sonos favorite named "{name}"'
```

Comment:
Please make the error translatable. See https://developers.home-assistant.io/docs/internationalization/core/#exceptions

Reply:
Thanks, did not know about that, updated.